.github/workflows/deploy-changed-cf.yaml

```diff
@@ -88,29 +88,105 @@ jobs:
         name: Set name of the CloudFormation stack
         run: |-
           echo "Ref is $GITHUB_REF"
-          BASE_NAME=biomage-$(basename $FILE_NAME | sed "s/\..*//")-$CLUSTER_ENV
-          echo "::set-output name=stack-name::$BASE_NAME"
+          BASE_NAME=$(basename $FILE_NAME | sed "s/\..*//")
+          STACK_NAME=biomage-$BASE_NAME-$CLUSTER_ENV
+
+          echo "::set-output name=file-name::$BASE_NAME"
+          echo "::set-output name=stack-name::$STACK_NAME"
         env:
           FILE_NAME: ${{ matrix.template }}
 
-      - id: deploy-template
-        name: Deploy CloudFormation template
-        if: ${{ !contains(matrix.template, 'irsa-') && github.ref == 'refs/heads/master' }}
+      - id: deploy-irsa-template
```

Comment:
This was previously placed below `deploy-template`. I switch this step before deploy template because now there are steps which act after `deploy-template`.